src/Umbraco.Web.UI.Client/src/views/propertyeditors/contentpicker/contentpicker.controller.js

```diff
@@ -469,26 +479,6 @@ function contentPickerController($scope, $q, $routeParams, $location, entityReso
 
     }
 
-    function setEntityUrl(entity) {
-
-        // get url for content and media items
-        if (entityType !== "Member") {
-            entityResource.getUrl(entity.id, entityType).then(function (data) {
-                // update url
-                $scope.renderModel.forEach(function (item) {
-                    if (item.id === entity.id) {
-                        if (entity.trashed) {
-                            item.url = vm.labels.general_recycleBin;
-                        } else {
-                            item.url = data;
-                        }
-                    }
-                });
-            });
-        }
-
-    }
-
```

Comment:
Should be fine to delete this as it's in a controller and (technically) not accessible outside the controller scope. I say technically because it's still possible in AngularJS to do some disgusting scope manipulation and get access to controller internals, but in reality it's a whole lot easier just to duplicate the function in your own code. 

If this were a service/factory, deleting would not be safe.

Reply:
Ah I see, good to know, I'll go ahead and resolve this then 😄